app/src/containers/WalletHome.vue

```diff
@@ -113,11 +166,17 @@ export default {
       this.$store.dispatch('forceFetchTokens')
     },
     initiateTransfer() {
-      this.$router.push({ path: '/wallet/transfer', query: { address: this.selected[0].tokenAddress.toLowerCase() } })
+      // this.$router.push({ path: '/wallet/transfer', query: { address: this.selected[0].tokenAddress.toLowerCase() } })
+      this.$router.push({ path: '/wallet/transfer' })
     },
     topup() {
       this.$router.push({ path: '/wallet/topup' })
     }
+  },
+  watch: {
+    finalBalancesArray(value) {
```

Comment:
can use this as a computed property instead. ideally use watchers only for async calls

Reply:
Done.. moved to computed property